internal/models/kubernetessecret/cluster/secretexport/secret_export_metod_get.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package secret
+
+import "github.com/go-openapi/swag"
+
+// VmwareTanzuManageV1alpha1ClusterNamespaceSecretexportGetSecretExportResponse Response from getting a SecretExport.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.cluster.namespace.secretexport.GetSecretExportResponse
+type VmwareTanzuManageV1alpha1ClusterNamespaceSecretexportGetSecretExportResponse struct {
+
+	// SecretExport returned.
+	SecretExport *VmwareTanzuManageV1alpha1ClusterNamespaceSecretexportSecretExport `json:"secretExport,omitempty"`
+}
+
+// MarshalBinary interface implementation
+func (m *VmwareTanzuManageV1alpha1ClusterNamespaceSecretexportGetSecretExportResponse) MarshalBinary() ([]byte, error) {
+	if m == nil {
+		return nil, nil
+	}
+	return swag.WriteJSON(m)
+}
+
+// UnmarshalBinary interface implementation
+func (m *VmwareTanzuManageV1alpha1ClusterNamespaceSecretexportGetSecretExportResponse) UnmarshalBinary(b []byte) error {
+	var res VmwareTanzuManageV1alpha1ClusterNamespaceSecretexportGetSecretExportResponse
```

Comment:
`secretexportGetSecretExport` can be `GetSecretExport`, same applied for the secrets

Reply:
Updated PTAL

Reply:
@Shubbhang351 this is a nit, you can keep it same as well. 

The reason being in the API model we get the full path to the request (auto generated via swagger gen)